utils/hare-bootstrap

```diff
@@ -209,8 +212,8 @@ $debug_p && set -x
 
 cdf=${1:-}
 
-if sudo systemctl --quiet is-active hare-consul-agent; then
```

Comment:
Isn't it safer just create a copy of `hare-bootstrap` script and modify it - so the new one will be used by Mini Provisioner, but we'll not break the existing behavior for debugging and testing?

@mssawant what do you think?

Reply:
And you mean we invoke the new one if `skip-consul-start` is specified to `hctl` interface? Although not for long but we will have to maintain multiple scripts and make sure that changes to one, if any, are incorporated in to other one too. On the other hand it will be cleaner to have a separate script.

Reply:
> And you mean we invoke the new one if skip-consul-start is specified to hctl interface? 

Not necessary. The script name will be different, so probably we'll not even need a flag.

Reply:
I am thinking how would we invoke that different script, do you mean we just move to the new script and `hctl bootstrap` command invokes only the new script? or do you suggest to have a new `hctl command` altogether that invokes the new script?

Reply:
No. We can leave `hare-bootstrap` script as is so everybody who have been using `hctl bootstrap` will still get just the same behavior they would expect.

But we can copy `hare-bootstrap` -> `hare-provision` and put @SwapnilGaonkar7's changes there. Then it will be invoked like this: `hctl provision --mkfs /my/cdf/file.yaml`

The latter script will not start Consul.

Thoughts?

Reply:
okay, so you are suggesting separate `hctl command`. I wonder if it would be confusing for the users of `hctl bootstrap`, we would need to document it well. In turn a flag like `skip-consul-start` seems more intuitive.
I think @SwapnilGaonkar7 is testing a "no change" option to hare-bootstrap (i.e. without `skip-consul-start`) May be we can just run `systemctl start hare-consul-agent` un-conditionally and handle the return code from systemd in case `hare-consul-agent` is already running.